certeasy-backend-app/src/main/java/org/certeasy/backend/certs/EmployeeCertSpec.java

```diff
@@ -0,0 +1,74 @@
+package org.certeasy.backend.certs;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.certeasy.backend.common.BaseCertSpec;
+import org.certeasy.backend.common.validation.ValidationPath;
+import org.certeasy.backend.common.validation.Validator;
+import org.certeasy.backend.common.validation.Violation;
+import org.certeasy.backend.common.validation.ViolationType;
+
+import java.util.Set;
+
+public class EmployeeCertSpec extends BaseCertSpec {
+
+    @JsonProperty("first_name")
+    private String firstName;
+    private String surname;
```

Comment:
do we go last_name or surname?
this because we have first_name and not just name

Reply:
Maybe we should switch to `name`

Reply:
Change has been made